kong/pdk/request.lua

```diff
@@ -652,7 +625,7 @@ local function new(self)
       error("header name must be a string", 2)
     end
 
-    return var["http_" .. replace_dashes(name)]
+    return http_get_header(name, ngx.ctx)
```

Comment:
Do not pass ctx here, so that it will not be cached.

Reply:
done.

Reply:
done.